### PR TITLE
Fix JRuby for ActiveRecord 6.0

### PR DESCRIPTION
**Pull request by keithdoggett**
Similar to #330, except I didn't have to modify the `postgis_connection` method. Just add the new adapter and update the tests. Also got rid of the travis CI and replaced it with a Github workflow.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          --name=postgres
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    env:
+      BUNDLE_GEMFILE: ${{matrix.gemfile}}
+    strategy:
+      matrix:
+        ruby: [3.0, 2.7, 2.6, 2.5, jruby]
```
Comment (BuonOmo) [inline .github/workflows/tests.yml:25]:
```suggestion
        ruby: ['3.0', '2.7', '2.6', '2.5', jruby]
```

lets follow the rules we've applied in rgeo/rgeo !

**Review by BuonOmo — COMMENTED** [review]
One review, also how is this PR not a merge conflict with the previous one Ia accepted ?

**Comment by keithdoggett** [comment]
> One review, also how is this PR not a merge conflict with the previous one Ia accepted ?

This one is going into the `6.0-stable` branch to add compatibility to older versions. I figured it'd be good to backport the fix since it's pretty much identical for ActiveRecord 5.2, 6.0, and 6.1.

**Review by BuonOmo — APPROVED** [review]
Then LGTM !

**Pull request merged**